systemd/manager.go

```diff
@@ -64,6 +60,13 @@ func NewSystemdUnitManager(uDir string) (*systemdUnitManager, error) {
 	return &mgr, nil
 }
 
+func createDbusConnection(systemdUser bool) (*dbus.Conn, error) {
```

Comment:
The indirection of this function seems unnecessary - I'd just inline the conditional into NewSystemdUnitManager